libutils/mutex.c

```diff
@@ -43,39 +43,39 @@ pthread_mutex_t *cft_server_children = &MUTEXES[3]; /* GLOBAL_T */
 pthread_mutex_t *cft_server_filter = &MUTEXES[4]; /* GLOBAL_T */
 
 
-int __ThreadLock(pthread_mutex_t *mutex,
+void __ThreadLock(pthread_mutex_t *mutex,
                  const char *funcname, const char *filename, int lineno)
 
 {
     int result = pthread_mutex_lock(mutex);
 
     if (result != 0)
     {
-        /* Log() is blocking on mutexes itself inside malloc(), so maybe not
-         * the best idea here. */
-        Log(LOG_LEVEL_ERR,
```

Comment:
I think the comment should stay in some form to explain why `Log` is not being used here.